scene/resources/particles_material.cpp

```diff
@@ -1416,7 +1416,7 @@ void ParticlesMaterial::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "spread", PROPERTY_HINT_RANGE, "0,180,0.01"), "set_spread", "get_spread");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "flatness", PROPERTY_HINT_RANGE, "0,1,0.01"), "set_flatness", "get_flatness");
 	ADD_GROUP("Gravity", "");
-	ADD_PROPERTY(PropertyInfo(Variant::VECTOR3, "gravity"), "set_gravity", "get_gravity");
+	ADD_PROPERTY(PropertyInfo(Variant::VECTOR3, "gravity", PROPERTY_HINT_NONE, "suffix:m/s"), "set_gravity", "get_gravity");
```

Comment:
This is wrong for two reasons:

* ParticlesMaterial is used for both 2D and 3D, it's just that the Z axis is ignored in 2D. For 2D this would be `px/s`, so we can't specify a unit here as it's wrong either way.
* If the above wasn't the case, gravity is in meters per second **squared**, not meters per second, so it should be `U"suffix:m/s\u00B2"`

Reply:
Ok, I'll remove the suffix altogether in this case.